api/rest.go

```diff
@@ -3492,22 +3505,66 @@ func handleUpdateSourceI18n(exec boil.Executor, id int64, i18ns []*models.Source
 		return nil, err
 	}
 
+	cu, errCU := models.ContentUnits(exec,
```

Comment:
CU for source should have no i18n. This sync is not needed. Client applications will show whatever they want knowing it's a SOURCE content unit

Reply:
fix